image-trigger

```diff
@@ -41,6 +41,7 @@ REFRESH = {
     "centos-8-stream": {},
     "centos-9-bootc": {},
     "centos-9-stream": {},
+    "centos-10": {},
```

Comment:
We're going to have to be careful about this if we start using the name "centos-10" in tests.  iirc, in testing farm tests we paste "-stream" back onto the end of the OS name if it's CentOS...

Reply:
Right, I'll adjust that in the PRs where I enable c10s. In our tests we mostly already do `.startswith('centos-9')` and such.

Reply:
See e.g. https://github.com/cockpit-project/starter-kit/pull/839